doc/source/whatsnew/v1.1.0.rst

```diff
@@ -42,7 +42,7 @@ Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
 - :class:`Styler` may now render CSS more efficiently where multiple cells have the same styling (:issue:`30876`)
--
+- :func:`Timestamp` now raises more explanatory error message when year, month or day is missing (:issue:`31200`)
```

Comment:
Thanks @fujiaxiang 

I don't know how important this is, but perhaps it should be :class:`Timestamp`?

Reply:
done, thanks